### PR TITLE
HaskellLintBear.py: Sort imports alphabetically

### PR DESCRIPTION
**Pull request by gs0510**
Used PyImportSortBear to sort the imports.

Closes https://github.com/coala/coala-bears/issues/355

### CODE REVIEW

File: bears/haskell/HaskellLintBear.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.DistributionRequirement import (
```
Comment (Adrianzatreanu) [inline bears/haskell/HaskellLintBear.py]: there should be no space in shortlog before colon

Comment (gs0510) [inline bears/haskell/HaskellLintBear.py]: Corrected the commit log.

File: bears/haskell/HaskellLintBear.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 from coalib.bearlib.abstractions.Linter import linter
```
Comment (Adrianzatreanu) [inline bears/haskell/HaskellLintBear.py]: also body shouldnt be in first person, try something like: PyImportSortBear was used to sort the imports correctly, could you :D ?

Comment (gs0510) [inline bears/haskell/HaskellLintBear.py]: Made the changes in the commit body :)

**Comment by Adrianzatreanu** [comment]
ack 47a7d19f17518abeea2eeb55e776e4f23e588ce2

**Comment by Adrianzatreanu** [comment]
this is good but requires a rebase as its out of date :) good work there, waiting for the rebase

**Comment by gs0510** [comment]
Rebased my repo.:)

**Comment by sils1297** [comment]
Reack https://github.com/gs0510/coala-bears/commit/51388e67df3ab90961a0a2c5bb6de35b1a2c2371

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
Did we mention our little survey yet? You could help us a great deal: coala.io/newform

**Pull request merged**